sendrecv/gst/webrtc-sendrecv.py

```diff
@@ -105,6 +106,25 @@ def on_incoming_stream(self, _, pad):
         decodebin.sync_state_with_parent()
         self.webrtc.link(decodebin)
 
+    def poll_cb(self):
+        done = False
+        while self.bus.peek() != None:
+            msg = self.bus.pop()
```

Comment:
You can directly pop or not? Without peek first

Reply:
I assume that I could, as poll_cb will get called again immediately if all messages haven't been read, not sure what's best?